scripts/Package_BGA/bga.py

```diff
@@ -177,7 +182,7 @@ def crtYdRound(x):
     f.append(Model(filename="{}Package_{}.3dshapes/{}.wrl".format(
                   config['3d_model_prefix'], package_type, fp_id)))
 
-    f.setTags("{} {} {}".format(package_type, balls, pitch_string))
+    f.setTags("{} {} {} {}".format(package_type, balls, pitch_string, additionalTag))
```

Comment:
This will put a space regardless of if the 4th argument is present or not.  Let's remove the space and format as I mentioned above.